wolfcrypt/src/asn.c

```diff
@@ -33283,6 +33323,64 @@ static int ParseCRL_Extensions(DecodedCRL* dcrl, const byte* buf,
             }
         #endif
         }
+        else if (oid == CRL_NUMBER_OID) {
+            localIdx = idx;
+            if (GetASNTag(buf, &localIdx, &tag, sz) == 0 &&
+                    tag == ASN_INTEGER) {
+                ret = GetASNInt(buf, &idx, &length, sz);
+                if (ret < 0) {
+                    WOLFSSL_MSG("\tcouldn't parse CRL number extension");
+                    return ret;
+                }
+                else {
+                    if (length > 1) {
+                    #ifdef WOLFSSL_SMALL_STACK
+                        mp_int* m;
+                    #else
+                        mp_int m[1];
+                    #endif
+                        int    i;
+
+                    #ifdef WOLFSSL_SMALL_STACK
+                        m = (mp_int*)XMALLOC(sizeof(*m), NULL,
```

Comment:
Please use `DYNAMIC_TYPE_BIGINT`. Make sure and fix the XFREE's below.

Reply:
Fixed